components/tidb_query_executors/src/simple_aggr_executor.rs

```diff
@@ -153,6 +177,13 @@ impl<Src: BatchExecutor> AggregationExecutorImpl<Src> for SimpleAggregationImpl
         let rows_len = input_logical_rows.len();
         self.has_input_rows |= rows_len > 0;
 
+        let n_bytes = self.n_bytes;
+
+        // FIXME: Are we counting this multiple times?
+        if rows_len > 0 && self.n_bytes == 0 {
+            self.n_bytes += rows_len * size_of::<usize>();
```

Comment:
where the new memory allocated?

Reply:
This is the vector of offsets, in this case the input_logical_rows.